### ISSUE TITLE
The Sidebar context menu shows tags when the experimental option is disabled. 

### ISSUE THREAD

**Issue by cinqmilleans** [files-community/Files#8451]
### Description

The Sidebar has a new menu to select visible sections. This menu always displays the Tags section. 

### Steps To Reproduce

1) Disable the experimental tags option in the options.
2) Open the Sidebar menu.
3) The Tag entry is visible. 

### Expected behavior

Entry should be hidden when experimental option is disabled.

### Files Version

Main branch

### Windows Version

Windows 11

### Relevant Assets/Logs

None

**Comment by gave92** [comment]
Even if you restart the app?

**Comment by gave92** [comment]
Check is [here](https://github.com/files-community/Files/blob/main/src/Files/Filesystem/FileTagsManager.cs#L51), should not add the section if disabled (but it's expected to only work after restart)
```cs
if (UserSettingsService.PreferencesSettingsService.AreFileTagsEnabled && UserSettingsService.AppearanceSettingsService.ShowFileTagsSection && section == null)
```
I'll che it out but I'm actually hoping we can remove the enabling switch in Experimantal altogether.

**Comment by cinqmilleans** [comment]
I'm talking about the context menu entry, which is still visible but doesn't do anything because the tags are hidden. But if the option will soon no longer be experimental, we can do nothing.
![SidebarMenu](https://user-images.githubusercontent.com/46631671/155022100-c81d3bb3-6ae9-4077-8f2a-1e2329d59f27.png)

**Comment by gave92** [comment]
Aaah that's true, thanks

**Comment by d2dyno1** [comment]
Closed with #8460
